### DISCUSSION TITLE
Alist无法在我的BCM4709C和MT7621构架路由上运行

### DISCUSSION THREAD

**Discussion by Hermit-Nameless** [AlistGo/alist#2191] [Q&A]
试了两台路由，斐讯K3（CPU是arm BCM4709C，梅林固件）和极路由B70（CPU是MT7621，Padavan固件)，下载了各arm和mipsle的版本，均无法在路由上运行
![Alist](https://user-images.githubusercontent.com/30073216/199636292-682c1746-5c11-4958-b3ff-704f42a3d229.jpg)

**Comment by mengguyi** [comment]
应该是`./alist server`没有sh

**Reply by Hermit-Nameless** [reply]
上面都有试过了，不行的，提示Illegal instruction和not found
非法指令可能是CPU架构不一样，那个not found我就不理解了

**Reply by uid-matthew** [reply]
> 上面都有试过了，不行的，提示Illegal instruction和not found 非法指令可能是CPU架构不一样，那个not found我就不理解了

老哥解决了么 我的MT7621也是一样的不行
